test/strings/types.jl

```diff
@@ -25,25 +25,51 @@ for i1 = 1:length(u8str2)
     end
 end
 
+# tests for SubString of a single multibyte `Char` string
+# we are consistent with `getindex` for `String`
+for idx in 0:1
+    @test SubString("∀", 1, idx) == "∀"[1:idx]
+end
+
+# invalid index of start of the index
```

Comment:
I don't understand that comment.

Reply:
I have tried to make it clearer now.

Reply:
Thanks. But I think "start index" should be "end index", right?

Reply:
Fixed. Only last one tested invalid start index.